src/main/java/org/spongepowered/api/event/world/ExplosionEvent.java

```diff
@@ -24,6 +24,9 @@
  */
 package org.spongepowered.api.event.world;
 
+import com.flowpowered.math.vector.Vector3i;
+import javafx.util.Pair;
```

Comment:
Swap for Tuple?, Ideally we don't want javafx imports

Reply:
Ah, okay. I've not used this sort of thing before. I'll try to fix it tomorrow morning.

Reply:
We have a tuple class here which is library non dependent it would be better to use this https://github.com/SpongePowered/SpongeAPI/blob/stable-7/src/main/java/org/spongepowered/api/util/Tuple.java